README.md

```diff
@@ -340,19 +239,24 @@ mvn clean test -P rest -Dcucumber.options="--tags @integration-ftp-ftp"
 
 #### Additional parameters
 
+##### Deploying Syndesis via the testsuite
+You can use profile `-P deploy` **with some other profile** (ie. mvn clean test -P deploy,ui) that sets the required parameters to clean the namespace and don't clean the namespace after tests.
```

Comment:
`clean the namespace and don't clean the namespace after tests`

I got confused the first time I read it even though I know what it means. Maybe `clean the namespace _before_ and don't clean the namespace after tests`?